rbc/omnisci_backend/omnisci_column.py

```diff
@@ -23,6 +23,10 @@ class OmnisciColumnType(OmnisciBufferType):
     """
     pass_by_value = True
 
+    @property
+    def buffer_extra_members(self):
+        return (('int32_t dict_id', (5, 8)),)
```

Comment:
Note for the future: wait until omnisci/omniscidb-internal#5492 lands in omniscidb dev image to revert this to (`5, 7)`